hw/ip/rstmgr/dv/rstmgr_cnsty_chk/rstmgr_cnsty_chk_sim_cfg.hjson

```diff
@@ -0,0 +1,44 @@
+// Copyright lowRISC contributors.
```

Comment:
Please add this to `hw//top_earlgrey/dv/top_earlgrey_sim_cfgs.hjson` so that this is picked up in nightly.

Reply:
I was wondering where to add this, thanks. Done.